bio/mehari/annotate-seqvars/wrapper.py

```
@@ -0,0 +1,88 @@
+__author__ = "Till Hartmann"
+__copyright__ = "Copyright 2025, Till Hartmann"
+__email__ = "till.hartmann@bih-charite.de"
+__license__ = "MIT"
+
+from snakemake.shell import shell
+import logging
+
+extra = snakemake.params.get("extra", "")
+
+keep_intergenic = str(snakemake.params.get("keep_intergenic", ""))
```

Ruff (0.11.9)

Undefined name `snakemake`

(F821)